Nice-HUD/Nice-HUD
# Feature/39 replace attack speed with bullet speed

## What
This PR adds the Replace attack speed with bullet speed. It includes the following changes:
- Changed the attack speed display method to a bullet speed display method in `DrawManagerImpl`.
- Updated `GameScreen` to display bullet speed instead of attack speed.
- adjust attack speed getter to bullet speed getter.

## Why
The display bullet speed feature can monitor bullet speed changes in real-time.

## Related Issue
Closes #39 

## How to Test
1. Start the application locally.
2. Check if the attack speed feature has been changed to the bullet speed feature.
3. Check if the bullet speed value increases when a bullet speed boost item is collected.

## Screenshots
<img width="742" alt="image" src="https://github.com/user-attachments/assets/89fc8b4f-7a47-4e82-890b-59568af470e2">
